examples/airflow/spark_conf/hive-site.xml

```diff
@@ -1,7 +1,7 @@
 <configuration>
    <property>
      <name>javax.jdo.option.ConnectionURL</name>
-     <value>jdbc:postgresql://airflow-postgres-1:5432/metastore_db</value>
+     <value>jdbc:postgresql://postgres:5432/metastore_db</value>
```

Comment:
Just curious: why did you have to change this?

Reply:
I was having trouble with new `docker compose` vs legacy `docker-compose` (which I think is docker compose V2 vs docker compose V1).

- The name `airflow-postgres-1` is just the container name, autogenerated by docker-compose. Since the service is defined as `postgres` in the `docker-compose.yml`, the [network name](https://docs.docker.com/compose/networking/) that should be referenced is `postgres`
- Something seems to have changed in the networking that `docker compose` sets up in newer versions. The name `airflow-postgres-1` was not reachable anymore, but `postgres` was. So I couldnt get the tests to run without this change